src/app/components/cds/inputs/TextArea.js

```diff
@@ -1,43 +1,31 @@
+/* eslint-disable */
 // DESIGNS: https://www.figma.com/file/rnSPSHDgFncxjXsZQuEVKd/Design-System?type=design&node-id=3606-26274&mode=design&t=ZVq51pKdIKdWZicO-4
-import React, { useState } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import TextField from './TextField';
 
 const TextArea = React.forwardRef(({
   autoGrow,
+  rows,
   maxHeight,
-  minHeight: minHeightProp,
   ...inputProps
 }, ref) => {
-  const [height, setHeight] = useState('auto');
-
-  const handleChange = (event) => {
-    const minHeight = minHeightProp || 0;
-    const newHeight = `${event.target.scrollHeight}px`;
-    // if the input is empty reset the height
-    if (event.target.value === '') {
-      setHeight('auto');
-    } else if ((autoGrow && event.target.scrollHeight >= minHeight) && (!maxHeight || event.target.scrollHeight <= maxHeight)) {
-      setHeight(newHeight);
-    }
-    if (inputProps.onChange) {
-      inputProps.onChange(event);
-    }
-  };
+  const handleChange = (event) => event.target.parentNode.dataset.replicatedValue = event.target.value;
```

Comment:
This is obviously the code the requires the eslint disable, a smart meedan engineer should be able to resolve it better than I.